notebooks/DrizzlePac/Initialization/Initialization.ipynb

```diff
@@ -0,0 +1,319 @@
+    "\n",
+    "An abbreviation for Astrometric Drizzle, AstroDrizzle was designed from the ground-up to substantially improve the handling of distortion in the image header World Coordinate System. AstroDrizzle removes geometric distortion, corrects for sky background variations, flags cosmic-rays, and combines images with optional subsampling. Drizzled data products from MAST are generated for single visit associations only.\n",
+    "\n",
+    "To combine data from additional visits, TweakReg may be used to update the image WCS using matched source lists. Once the full set of images of a given target are properly aligned, they may be combined using AstroDrizzle.\n",
+    "\n",
+    "While the DrizzlePac software has been optimized to work with Hubble Space Telescope (HST) data, it can work with other types of data so long as the images adhere to the FITS standards for multi-extension files and for describing the World Coordinate System (WCS). It assumes that all distortions have been properly described in the WCS of the image, e.g. via the SIP distortion coefficients. [More details may be found here](http://www.stsci.edu/hst/HST_overview/drizzlepac/adfeatures) under the section 'Aligning to Non-HST Image'. \n",
+    "\n",
+    "In this notebook, we will demonstrate how to set up your environment to analyze HST data with links to several resources, as well as a demonstration of how to download observations from the HST archive and their associated reference files."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<div class='alert-info'>Before working with HST data, users are advised to consult both the data handbooks and the instrument team websites for the instrument of interest. Additional useful discussion of the drizzling algorithm and of how distortion information is represented in the image header may be found in the <a href=\"http://documents.stsci.edu/hst/HST_overview/documents/DrizzlePac/DrizzlePac.cover.html\">DrizzlePac Handbook</a>.</div>"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "**Notes about HST files:**\n",
+    "\n",
+    "HST images are stored using the Flexible Image Transport System (FITS). In particular, HST data use a multi-extension FITS (MEF) format. In this format, science data, error arrays, and data quality information for a single observation are stored as different extensions within a single file. The data format for the detector you are working with may be found in your instrument's data handbook, e.g. for [WFC3](http://www.stsci.edu/hst/wfc3/documents/handbooks/currentDHB/wfc3_cover.html) and [ACS](http://www.stsci.edu/hst/acs/documents/handbooks/currentDHB/acs_cover.html).\n",
+    "\n",
+    "\n",
+    "\n",
+    "**Good data quality flags:**\n",
```

Comment:
This whole section seems out-of-context.  If I'm coming to these notebooks for the first time I don't know anything about this.  Particularly given that it's an "In april 2018" thing.  I'd say it either need more description, or should be moved to notebooks that reference particular data sets where it applies?

Or alternatively, a single sentence that says something like "here there be dragons.  See the ACS and WFC3 data handbook for details on those intruments."

Reply:
@tddesjardins - I'll need you input regarding whether we want to update this section or leave it as is.